lib/modules/platform/gitlab/index.ts

```diff
@@ -666,15 +673,9 @@ async function tryPrAutomerge(
       ];
       const desiredStatus = 'can_be_merged';
       // The default value of 5 attempts results in max. 13.75 seconds timeout if no pipeline created.
-      const retryTimes = parseInteger(
-        process.env.RENOVATE_X_GITLAB_AUTO_MERGEABLE_CHECK_ATTEMPS,
-        5,
-      );
+      const retryTimes = config.gitlabAutoMergeableCheckAttempts ?? 5;
```

Comment:
```suggestion
      const retryTimes = config.gitlabAutoMergeableCheckAttempts!;
```

they will be set from config at runtime, so please also set them in tests.